client/components/data/query-active-theme/README.md

```diff
@@ -0,0 +1,42 @@
+Query Theme
+============
+
+Query Active Theme is a React component used in managing the fetching sites active theme data: themeId, themeName and themeCost object.
```

Comment:
Typo; should probably be something like `the fetching of a given site's active theme ID`. And let's just drop the mention of `themeName` and `themeCost`, I can't see us using them much going forward.